docs/octopus-rest-api/octopus-cli/build-information.md

```diff
@@ -8,7 +8,9 @@ Pushes build information to Octopus Server.
 **build-information options**
 
 ```text
-Usage: Octo build-information [<options>]
+Pushes build information to Octopus Server.
```

Comment:
The help output now includes the same text as the index page does.
Might be worth a bit of a review to make sure the pages still make sense